src/components/Password/Password.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { render, screen } from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import { Password } from './Password';
+
+const ControlledPassword = props => {
+    const [value, setValue] = useState('');
+    const handleChange = e => {
+        setValue(e.target.value);
+    };
+
+    return <Password {...props} id="password-id" value={value} onChange={handleChange} />;
+};
+
+describe('Password', () => {
```

Comment:
How do we know this will work properly when using like:

```js
<form>
    <Password name="password" />
    <button type="submit">Submit</button>
</form>
```

I'd expect on submitting the form I'd be able to get the value of the password.  `new FormData(form)` as we are expected from any input. What are your thoughts on validating that? 

https://codesandbox.io/s/wave-form-s83d3?file=/src/App.tsx

> It probably works as we are using `Input` component under the hood

Reply:
We can validate this, but technically we will validate HTML spec 🤷  If you think it is valuable I don't see any issues adding this check